### PR TITLE
Release Kaptain 1.2.0-1.1.0

### PR DESCRIPTION
**Pull request by akirillov**
## Jira Ticket
[D2IQ-74822: site docs for the new Kaptain release](https://jira.d2iq.com/browse/D2IQ-74822)

## Description of changes being made
A new version of the site docs for Kaptain Release

## Checklist
- [x] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [x] Test all commands and procedures, if applicable.
- [x] Create your PR against `main`. 
- [x] Update all links if you are moving a page.
- [x] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/kaptain/1.2.0-1.1.0/release-notes/1.1/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+# Release Notes
+
+## 1.2.0-1.1.0 - Released 12 May 2021
+
+This document describes the new features, caveats, and resolved issues in D2iQ Kaptain.
+
+### New features
+
+* Upgradability and configuration updates of existing Kaptain installations without user workloads interruption
```
Comment (ianhellstrom) [inline pages/dkp/kaptain/1.2.0-1.1.0/release-notes/1.1/index.md:18]:
```suggestion
* Full zero-downtime upgradability of Kaptain and all of its configurations
```

Comment (ianhellstrom) [reply pages/dkp/kaptain/1.2.0-1.1.0/release-notes/1.1/index.md:18]: The top feature should be a bit punchier (as long as it's accurate).

**Review by cmastr — APPROVED** [review]

**Review by cgawron1 — APPROVED** [review]
lgtm

File: pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+The minimum total number of disks required for MinIO in distributed mode is 4 (same as minimum disks required for 
+[erasure coding](https://docs.min.io/docs/minio-erasure-code-quickstart-guide)).  
+`minioStorageCapacity` - the total capacity of a single MinIO server pool.
+<div style="color: #8a6d3b; background-color: #fcf8e3; border-color: #faebcc; padding: 15px; margin-top: 10px; margin-bottom: 10px; border: 1px solid transparent; border-radius: 4px;">
+<p class="message--warning"><strong>WARNING: </strong>By default, MinIO shards the objects across N/2 data and N/2 parity 
+drives, therefore it is recommended to specify `minioStorageCapacity` value twice as much as planned for use.</p>
+</div>
+
+ To change the default MinIO cluster settings, create a configuration file `parameters.yaml` with the parameters 
+ described above:
```
Comment (cneth) [inline pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md:39]:
```suggestion
 To change the default MinIO cluster settings, create a configuration file `parameters.yaml` with the properties described above:
```

File: pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+```yaml
+minioServers: 4
+minioVolumesPerServer: 4
+minioStorageCapacity: 160Gi
+```
+This configuration will deploy 4 MinIO servers, each server will get 4 volumes with 10Gi capacity per volume.  
+To learn more about MinIO distributed mode and replication see the [official documentation](https://docs.min.io/docs/).
```
Comment (cneth) [inline pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md:46]:
```suggestion
This configuration will deploy 4 MinIO servers; each server will use 4 volumes with 10Gi capacity per volume.  
To learn more about MinIO distributed mode and replication see the [official documentation](https://docs.min.io/docs/).
```

File: pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+To install Kaptain with the provided parameters, run the following command on the cluster:
+```bash
+kubectl kudo install ./kubeflow-1.2.0_1.1.0.tgz \
+      --instance kaptain \
+      -P parameters.yaml \
+      --namespace kubeflow \
+      --create-namespace
+```
+
+## Expanding MinIO cluster
```
Comment (cneth) [inline pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md:57]:
```suggestion
## Expanding the MinIO cluster
```

File: pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+MinIO cluster can be expanded by increasing the number of server pools in the cluster.
```
Comment (cneth) [inline pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md:59]:
```suggestion
The MinIO cluster can be expanded by increasing the number of server pools in the cluster. 
```

File: pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+      --namespace kubeflow \
+      --create-namespace
+```
+
+## Expanding MinIO cluster
+
+MinIO cluster can be expanded by increasing the number of server pools in the cluster. 
+<div style="color: #8a6d3b; background-color: #fcf8e3; border-color: #faebcc; padding: 15px; margin-top: 10px; margin-bottom: 10px; border: 1px solid transparent; border-radius: 4px;">
+<p class="message--warning"><strong>WARNING: </strong>Extending an existing server pool is not supported. The cluster 
+can only be scaled linearly, by using the initial server pool as a base scale unit.</p>
```
Comment (cneth) [inline pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md:62]: I find this wording confusing. How about this?

```suggestion
<p class="message--warning"><strong>WARNING: </strong>Expand an existing server pool is not supported. The cluster can only by adding additional server pools.</p>
```

File: pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+</div>
+
+To update already installed Kaptain instance with the provided parameters, run the following command on the cluster:
+```bash
+kubectl kudo update --instance kaptain --namespace kubeflow -p minioServerPools=2
+```
+This command will update the existing Kaptain instance and double up MinIO cluster capacity with almost no downtime.
```
Comment (cneth) [inline pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md:69]: Can we be more precise here about the downtime?  "Almost no downtime" doesn't sounds like "no downtime".  Should we add a note (like we have in the MySQL pages) about what can happen to clients during this change?

```suggestion
This command will update the existing Kaptain instance and double the MinIO cluster capacity with almost no downtime.  
```

File: pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+## Known limitations
+1. Data redundancy is only guaranteed within the single server pool - if more than a half of the drives in a pool 
+are lost, the data will not be reconstructed.
+2. When the MinIO cluster is expanded by adding the new server pool, the data from the initial pool will not be 
+replicated/rebalanced between the existing and the new pool.
```
Comment (cneth) [inline pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md:75]:
```suggestion
1. Data redundancy is only guaranteed within a single server pool - if more than a half of the drives in a pool are lost, the data cannot be reconstructed.
2. When the MinIO cluster is expanded by adding a new server pool, the data from the initial pool will not be replicated/rebalanced between the existing and the new pool.
```

File: pages/dkp/kaptain/1.2.0-1.1.0/troubleshooting/kaptain-1.1.0/index.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
+---
+layout: layout.pug
+navigationTitle: Troubleshooting Kaptain 1.0
+title: Troubleshooting Kaptain 1.0
+menuWeight: 10
+excerpt: View troubleshooting information for Kaptain 1.1.0
```
Comment (cneth) [inline pages/dkp/kaptain/1.2.0-1.1.0/troubleshooting/kaptain-1.1.0/index.md:6]: This page is already 'versioned' higher up, do we need to include the version here as well?  It is confusing (and makes more work for us when releasing a new version) to have the version number in the title of the page here.

```suggestion
navigationTitle: Troubleshooting Kaptain
title: Troubleshooting Kaptain 
menuWeight: 10
excerpt: View troubleshooting information for Kaptain
```

**Review by cneth — CHANGES_REQUESTED** [review]

**Comment by akirillov** [comment]
thanks for the thorough review, @cneth! applied your suggestions.

File: pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md [hunk 10]
```diff
@@ -54,22 +49,19 @@ kubectl kudo install ./kubeflow-1.2.0_1.1.0.tgz \
...
 
-## Expanding MinIO cluster
+## Expanding the MinIO cluster
 
-MinIO cluster can be expanded by increasing the number of server pools in the cluster. 
+The MinIO cluster can be expanded by increasing the number of server pools in the cluster. 
 <div style="color: #8a6d3b; background-color: #fcf8e3; border-color: #faebcc; padding: 15px; margin-top: 10px; margin-bottom: 10px; border: 1px solid transparent; border-radius: 4px;">
-<p class="message--warning"><strong>WARNING: </strong>Extending an existing server pool is not supported. The cluster 
-can only be scaled linearly, by using the initial server pool as a base scale unit.</p>
+<p class="message--warning"><strong>WARNING: </strong>Expand an existing server pool is not supported. The cluster can only by adding additional server pools.</p>
```
Comment (cneth) [inline pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md:56]: Sorry, seems like there was some kind of typo in my previous comment! My apologies!

```suggestion
<p class="message--warning"><strong>WARNING: </strong>Expanding an existing server pool is not supported. The cluster can only by expanded by adding additional server pools.</p>
```

**Review by cneth — CHANGES_REQUESTED** [review]
Looks good!  I found one minor issue in one of my previous suggestions.  I'll approve after that is applied!

**Review by cneth — APPROVED** [review]

**Pull request merged**